### PR TITLE
Popup menu for mobile portfolio.

### PR DESCRIPTION
**Pull request by telanet**
#Mobile menu milestones
Install JavaScript linters
When the hamburger button is clicked the mobile menu appears.
When the close (X) button, is clicked the mobile menu disappears.
When any of the mobile menu options, is clicked the mobile menu disappears.
When the mobile menu options, are clicked a correct part of the page is displayed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,55 +1,101 @@
-# Project Name
+# HTML & CSS3 & JavaScript Course
```
Comment (elyor-doniyorov) [inline README.md]: - [ ] Please remove current Readme and build your Readme from scratch with the project title and description, screenshots, live demo link, Installation and setup section, author information, github repo issues page link and license.

File: index.html [hunk 2]
```diff
@@ -15,19 +23,24 @@
...
+      <li><a href="#" class="link link-text fas fa-envelope"></a></li>
+        <li class="cancel-btn">
+          <i class="fas fa-times"></i>
+        </li>
+    </ul>  
+  </div>
+  <div class="menu-btn">
+    <i class="fas fa-bars"></i>
+  </div>
+</nav>
```
Comment (elyor-doniyorov) [inline index.html]: - [ ] Your current mobile menu does not work properly. For example, when I click any of the mobile menu options, a correct part of the page is not displayed.
- [ ] Moreover, your current design doesn't match the original design. Kindly fix it 🙏 
- Original:
Screenshot (175)
- Your design:
Screenshot (174)

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1,27 +1,20 @@
...
 
 - HTML
 - CSS
+- Javascript
 
-# Live Demo
-
-- https://telanet.github.io/Portfolio/
-
+## To get a local copy up and running follow these simple example steps.
```
Comment (FrederickMih) [inline README.md:17]: - [OPTIONAL] Please let's add these steps to make our REAdME more professional.

File: index.html [hunk 4]
```diff
@@ -15,19 +23,23 @@
...
-        <ul class="menu-list ul-list horizontal-list">
-            <li class="list-item-text"><a href="#" class="link link-text">Portfolio</a></li>
-            <li class="list-item-text"><a href="#" class="link link-text">About</a></li>
-            <li class="list-item-text"><a href="#" class="link link-text">Contact</a></li>
-            <li class="list-item-text"><a href="#" class="link link-text fas fa-envelope"></a></li>
-         </ul>
-        <div class="hamburger-menu" id="hamburger-menu"><i class="fas fa-bars"></i>
-      </div>
-    </nav>
-</header>
```
Comment (FrederickMih) [inline index.html:30]: - [ ] Nice job for implementing some changes requested by the previous reviewer :+1:. However the font, colors margins are not pretty much the same as those in Figma. Once more, let's work on this again.

Your Implementation | Figma Design
:---:|:---:
 |

**Review by FrederickMih — CHANGES_REQUESTED** [review]
### Hi Team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @telanet 👋🏻, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**